elixir/wrapper/optimizer.py

```diff
@@ -0,0 +1,227 @@
+import math
+from collections import defaultdict
+from enum import Enum
+from typing import Dict, Set, Tuple
+
+import colossalai.nn.optimizer as ColoOptimizer
```

Comment:
As a module is imported, a lower case module name should be used instead of a class-like name.

```
import colossalai.nn.optimizer as colo_optimizer
```

Reply:
done.